airflow/contrib/hooks/grpc_hook.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from airflow.hooks.base_hook import BaseHook
+from airflow.exceptions import AirflowException
+import grpc
+from google import auth as google_auth
+from google.auth import jwt as google_auth_jwt
+from google.auth.transport import grpc as google_auth_transport_grpc
+
+class GrpcHook(BaseHook):
+	"""
+	General interaction with GRPC servers.
+	"""
+
+	def __init__(self, grpc_conn_id, interceptors=[], custom_connection_func=None):
```

Comment:
It's missing a call to `__init__()` of the super class.

Reply:
For this I think it's not necessary as the BaseHook doesn't really do init and all the other hooks are not calling super if they are based on BaseHook